swo_aws_extension/parameters.py

```
@@ -106,3 +128,38 @@ def get_responsibility_transfer_id(source: dict[str, Any]) -> str | None:
+    updated_order = set_order_parameter_visibility(
+        updated_order,
+        OrderParametersEnum.MASTER_PAYER_ACCOUNT_ID.value,
+        visible=bool(get_mpa_account_id(order)),
+    )
+    updated_order = set_order_parameter_visibility(
+        updated_order,
+        OrderParametersEnum.ORDER_ACCOUNT_NAME.value,
+        visible=bool(get_order_account_name(order)),
+    )
+
+    return updated_order  # noqa: RET504
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `RET504`)

Remove unused `noqa` directive

(RUF100)